src/VCreditCard.vue

```diff
@@ -22,43 +22,64 @@
             <div class="vcc-col">
                 <div class="vcc-form-group">
                     <label class="vcc-label" for="holder">{{$t('form.holder')}}</label>
-                    <input type="text" class="vcc-control" id="holder" v-model="fullName" autocomplete="cc-name"/>
+                    <input type="text" class="vcc-control"
+                           :data-error="(cardErrors.cardFullName)"
+                           id="holder" v-model="fullName" autocomplete="cc-name"/>
+                    <div v-if="cardErrors.cardFullName" class="error">
+                        <small>{{ $t('validation.cardFullName') }}</small>
+                    </div>
                 </div>
             </div>
             <div class="vcc-col">
                 <div class="vcc-form-group">
                     <label class="vcc-label" for="number">{{$t('form.number')}}</label>
-                    <card-number-field class="vcc-control" v-model="cardNumber" id="number">
-                    </card-number-field>
+                    <card-number-field :data-error="(cardErrors.cardNumber)"
+                                       class="vcc-control" v-model="cardNumber" id="number"/>
+                    <div v-if="cardErrors.cardNumber" class="error">
```

Comment:
You should also need to check if user wants to display validation errors and validation is enabled
```html
<div v-if="validatorEnabled && validatorFields.cardNumber && displayValidationErrors && cardErrors.cardNumber" class="vcc-input-error">
...
```